src/arno_evap.c

```diff
@@ -85,6 +85,7 @@ double arno_evap(layer_data_struct *layer,
   double max_infil;
   double Evap;
   double tmpsum;
+  double rarc_soil = 100.0;
```

Comment:
Gives better results than 0 or 200.  I didn't do a fancy Bayesian parameter
estimation, not enough time for that.

I apologize for making this a hard-wired value - I considered making it
another input soil parameter, but we have so many of those already.  And
this likely won't noticeably affect anyone who has been using the
traditional VIC set-up, since bare soil rarely occurs in the traditional
parameters.

On Mon, Nov 30, 2015 at 12:35 PM, Joe Hamman <notifications@github.com>
wrote:

> In src/arno_evap.c
> <https://github.com/UW-Hydro/VIC/pull/309#discussion_r46199555>:
>
> > @@ -85,6 +85,7 @@ double arno_evap(layer_data_struct *layer,
> >    double max_infil;
> >    double Evap;
> >    double tmpsum;
> > +  double rarc_soil = 100.0;
>
> Why 100.0?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Hydro/VIC/pull/309/files#r46199555>.
>